decidim-core/app/presenters/decidim/metric_charts_presenter.rb

```diff
@@ -40,19 +40,32 @@ def render_not_highlighted(not_highlighted_metrics)
 
     def render_metrics_data(metric_name, opts = {})
       content_tag :div, class: opts[:klass].presence || "column medium-6" do
-        content_tag :div,
-                    "",
-                    id: "#{metric_name}_chart",
-                    class: "areachart metric-chart #{opts[:graph_klass]}",
-                    data: {
-                      chart: "areachart",
-                      metric: metric_name,
-                      info: {
-                        title: I18n.t("decidim.metrics.#{metric_name}.title"),
-                        object: I18n.t("decidim.metrics.#{metric_name}.object")
-                      }
-                    }
+        render_join_description(opts) if opts[:axis]
```

Comment:
Don't use axis because it's not understandable.
Not using concat to join blocks.